ci-operator/jobs/open-cluster-management/registration/open-cluster-management-registration-release-2.1-postsubmits.yaml

```diff
@@ -69,6 +69,8 @@ postsubmits:
           value: registration
         - name: COMPONENT_VERSION
           value: 0.0.2
+        - name: PIPELINE_MANIFEST_REMOTE_REPO
+          value: registry.svc.ci.openshift.org/open-cluster-management
```

Comment:
shouldn't this be quay.io/open-cluster-management?

Reply:
No. At the time the manifest is written, the image only exists in `registry.svc.ci.openshift.org`.

This value is used to trigger the retag to copy the image into quay during this step:

https://github.com/open-cluster-management/pipeline/blob/quay-retag/.travis.yml#L36

Reply:
alright, i am not fully understand the intention to retag here though.